site/develop/committers.md

```diff
@@ -46,6 +46,7 @@ Sergio Pena             | spena        | Committer
 Siddharth Seth          | sseth        | Committer
 Stephen Walkauskas      | swalkaus     | Committer
 Vaibhav Gumashta        | vgumashta    | Committer
+William Hyun            | william      | Committer
```

Comment:
Order is alphabetical so it should probably be after Wei but feel free to merge after that

Reply:
The order is alphabetical in ID, isn't it?

Reply:
It seems that ID-order.
```
Gang Wu                 | gangwu       | PMC
Alan Gates              | gates        | PMC
```

Reply:
ID-order looks dominant. We had better fix the sorting in a separate PR, @williamhyun and @pgaref .
```
Matthew McCline         | mmccline     | Committer
Sandeep More            | moresandeep  | Committer
Naveen Gangam           | ngangam      | Committer
```